Gemfile

```diff
@@ -9,8 +9,6 @@ gemspec
 group :testapp do
   gem 'bootsnap',     '>= 1.1.0', require: false
   gem 'listen'
-  gem 'kramdown'
-  gem 'RedCloth'
```

Comment:
I noticed we don't need these since the rich types were ripped out in the base branch.